.github/workflows/test.yml

```diff
@@ -62,6 +62,30 @@ jobs:
         env:
           PERCY_TOKEN: ${{ secrets.PERCY_TOKEN }}
 
+  chromatic-deployment:
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        node-version: [14.x]
+
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v2
+        with:
+          fetch-depth: 0 # 👈 Required to retrieve git history
+      - name: Install dependencies
+        run: yarn install --immutable
+      - name: Build Storybook
+        run: yarn workspace @chanzuckerberg/eds-components run build:storybook
```

Comment:
In the Percy job, we were using `yarn workspaces foreach run build:storybook` but Storybook only exists in `components`, so went with this instead. Are we expecting more packages under this monorepo and should I make it the `foreach` instead?

Reply:
We can always update later, so this seems fine.